### ISSUE TITLE
[2pt] Add columns to existing tables in SQL

### ISSUE THREAD

**Issue by TarantoolBot** [tarantool/doc#1650]
# Add columns to existing tables in SQL

**Product:** Tarantool (SQL)
**Since:** 2.7.1
**Audience/target:** Tarantool users
**Root document:** https://www.tarantool.io/en/doc/latest/reference/reference_sql/sql_statements_and_clauses/#alter-table
**SME:** @romanhabibov
**Peer reviewer:** @

# Details

Now, it is possible to add columns to existing empty spaces using
<ALTER TABLE table_name ADD [COLUMN] column_name column_type ...>
statement. The column definition is the same as in <CREATE TABLE>
statement.

* Space emptiness is Tarantool's restriction. Possibilty to add
column to non empty space will be implemented later.

For example:

```
tarantool> box.execute("CREATE TABLE test (a INTEGER PRIMARY KEY)")
---
- row_count: 1
...

tarantool> box.execute([[ALTER TABLE test ADD COLUMN b TEXT
         >                                           CHECK (LENGTH(b) > 1)
         >                                           NOT NULL
         >                                           DEFAULT ('aa')
         >                                           COLLATE "unicode_ci"
         >             ]])
---
- row_count: 1
...
```
Requested by @romanhabibov in https://github.com/tarantool/tarantool/commit/6cff9025909529febeff2f04bf714f70bd3a79e1.

**Comment by pgulutzan** [comment]
@romanhabibov: This is only in version 2.7, right?

**Comment by Totktonada** [comment]
Yep, since 2.7.0-38-g6cff90259.

**Comment by pgulutzan** [comment]
I changed version 2.6 adding a note that there is a plan to allow more. I will change version 2.7 when it exists.

**Comment by veod32** [comment]
> I changed version 2.6 adding a note that there is a plan to allow more. I will change version 2.7 when it exists.

Now, the 2.6 documentation branch is not in the use. There will be no 2.7 branch, so all the updates should be done in the `latest` branch which is the default one now. As I can see, your [commit](https://github.com/tarantool/doc/commit/7cd2c5c701cf807f7948c26a4dcb41a4978d340a) is already in the `latest` branch.

I think you can make final updates to the documentation according to this issue description: functional changes are not planned, they are already implemented in the 2.7.1 release. After making doc updates, please ask @NickVolynkin or @veod32  for the peer review of your pull request.
